lib/sipInterconnect.js

```diff
@@ -30,40 +30,6 @@ function SipInterconnect(config, properties) {
       this[key] = properties[key];
     }
   }
-
```

Comment:
Is this being removed because its superseded by https://github.com/opentok/opentok-node/blob/e088b4167a88edad77bddc36a32e078ef3c9c84e/lib/opentok.js#L377 ?